mainwindow.cpp

```diff
@@ -270,7 +269,7 @@ void MainWindow::fillOutlineEnable(bool enable)
 
 void MainWindow::bridgesAvailable()
 {
-    bool bridgesEnabled = ui->vectorialCheckBox->isChecked() || ui->optimiseCheckBox->isChecked();
+    bool bridgesEnabled = ui->vectorialCheckBox->isChecked() || true;

```

Comment:
Or true?

Reply:
The documentation is a bit confusing
from https://github.com/pcb2gcode/pcb2gcode/wiki/Manual

    --bridges unit
    add bridges with the given width to the outline cut. --bridgesnum bridges will be created for each outline 
    closed line. This option requires --optimise

I'm in doubt if the bridges generating still needs --optimize 
(or --vectorial for that matter)
If only --vectorial is required I can just delete the:

     || true

Reply:
if --optimize still is required in pcb2gcode I'll have to figure out testing for a non zero double instead of the or true

Reply:
That text is old and I didn't write it. Bridges doesn't require vectorial. Vectorial is mandatory anyway. Bridges doesn't require optimise, either.

The number is how many Bridges to put on each outline cut. The width is how wide each bridge should be. Bridges are places where the outline cut is modified so that it doesn't fully disconnect your board from the PCB. Sort of like making a perforation so that you can disconnect your board from the PCB when you are done.

I hope that makes sense! I'll look over the pcb2gcode help text at some point and fix it up. The man page will need work, too, though maintaining both the man page and the wiki has been a pain and I'd rather just maintain the wiki because I can add graphics and stuff there.